azavea/cac-tripplanner
# Bugfix/kak/fix carousel console error#1118

## Overview

Fixes a console error on detail pages that have only one image. Also adds to the default development data, which should help with testing the console error, and includes package updates.


### Demo

Summary card for the default article:

![image](https://user-images.githubusercontent.com/960264/61312425-6288b380-a7c6-11e9-8680-f98b36e837b8.png)

Multiple images for a default destination:

![image](https://user-images.githubusercontent.com/960264/61312477-7d5b2800-a7c6-11e9-9860-6f891251ccba.png)


## Testing Instructions

 * Reprovision `app` VM to copy images and run migrations
 * There should be a default article added
 * There should be a default event added
 * John Heinz National Wildlife Refuge and Bartram's Garden detail pages should show multiple images in carousel
 * Other place detail pages (without the carousel) should have no console error (as described in #1118)


Closes #1104 
Closes #1118
